README.md

```diff
@@ -988,6 +988,26 @@ kafka = Kafka.new(
 )
 ```
 
+##### OAUTHBEARER
+This mechanism is supported in kafka >= 2.0.0 as of [KIP-255](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=75968876)
+
+In order to authenticate using OAUTHBEARER, you must set the client with an instance of a class that implements a `token` method (the interface is described in `lib/kafka/sasl/oauth.rb`) which returns an ID/Access token.
```

Comment:
I'm not an expert on GitHub Readme Markdown, but I assume there's some way to make this a relative link to the file, so the reader can just click it, rather than finding it themselves.